extensions/attention-stream/package.json

```diff
@@ -2,8 +2,10 @@
   "name": "rally-attention-stream",
   "version": "0.4.6",
   "scripts": {
-    "build": "cp manifest.prod.json manifest.json && npm run build:glean && rollup -c 2>&1 && npm run tailwind && npm run package:webext",
-    "dev": "cp manifest.dev.json manifest.json && npm run build:glean && npm run clean && npm run tailwind && npm run dev:extension",
+    "config:chrome": "tsx scripts/generate-manifest.ts --browser=chrome --manifest-version=3",
+    "config:firefox": "tsx scripts/generate-manifest.ts --browser=firefox --manifest-version=2",
+    "build": "npm run config:chrome && npm run build:glean && rollup -c 2>&1 && npm run tailwind && npm run package",
+    "dev": "npm run build:glean && npm run clean && npm run tailwind && npm run dev:extension",
```

Comment:
It also looks like the `dev` target no longer gets a `manifest.json` out of the box. Shouldn't this be prefixed with `npm run config:chrome` just like the regular `build` target?

Reply:
Good catch thanks